### DISCUSSION TITLE
Artist Compilations are annoying

### DISCUSSION THREAD

**Discussion by LinuxBoy-96** [strawberrymusicplayer/strawberry#1226] [Polls]
I would like to share my biggest problem, and it's probably my one too.

I have 465 songs at the moment in my library and a lot of them are in a "Best of" type of album.
All these song don't appear in the speciffic band folder.

For exemple, I have 3 Sabaton songs that are not with the rest of my Sabaton song, because they come from an album where there is also already released songs...

Also, I have some song that are their without any explanations. Not a "Best of" album, not aven multiple artist on that album. I'm puzzled.

82 of my song are in this section.

**Comment by trincowski** [comment]
I'm new here, so don't find strange that this answer is a year late... but this problem is fixed now, right? Do these discussions ever close, after issues or suggestions are implemented (or even dismissed)?

**Reply by jonaski** [reply]
I suggest to use the official forum which is https://forum.strawberrymusicplayer.org/
This project has only 1 developer, me, and there isn't anyone else here that answers questions from users. I do not participate here since we already got a community forum outside of GitHub that works better. In the long run I'm going to attempt to move everything away from GitHub.
